### DISCUSSION TITLE
Wrong fullyQualifiedName is returned from return type declaration

### DISCUSSION THREAD

**Discussion by pavel163** [LemonAppDev/konsist#821] [Report a Problem]
### Tell us what version of Konsist you use.

0.13.0

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the problem

Hello

I have such a test:

```kotlin
Konsist
    .scopeFromProject()
    .interfaces()
    .withNameEndingWith("UseCase")
    .functions()
    .withName("execute")
    .assertTrue {
        it.hasReturnType { typeDeclaration ->
            typeDeclaration.fullyQualifiedName == "Result"
            }
        }
```

And UseCase:
```kotlin
package com.example.konsist

interface SendSmileResultUseCase {

    suspend fun execute(userId: String, providerId: String, accountId: String): Result<Unit>
}
```

When I run the test, it fails because `typeDeclaration.fullyQualifiedName` returns `com.example.konsist.SendSmileResultUseCase`

but if I replace the name with `SendSmileUseCase`, the test works fine

### Actual behaviour

**Result** in the name of class affects typeDeclaration.fullyQualifiedName

### Expected behaviour

**Result** in the name of class doesn't affect typeDeclaration.fullyQualifiedName

**Comment by jibidus** [comment]
It seems a duplicate of #738

**Reply by igorwojda** [reply]
Fixed in https://github.com/LemonAppDev/konsist/pull/830

**Comment by nataliapeterwas** [comment]
A new version v0.14.0 has been released. It contains the fix.
See [v0.14.0 release](https://github.com/LemonAppDev/konsist/releases/tag/v0.14.0)
